packages/blockchain-wallet-v4-frontend/src/scenes/Signup/index.tsx

```diff
@@ -145,8 +139,7 @@ const mapStateToProps = (state: RootState): LinkStatePropsType => ({
   search: selectors.router.getSearch(state) as string,
   signupCountryEnabled: selectors.core.walletOptions
     .getFeatureSignupCountry(state)
-    .getOrElse(false) as boolean,
-  userGeoData: selectors.auth.getUserGeoData(state) as GeoLocationType
```

Comment:
where does this come from? can we delete all of the logic here including selectors and api calls?